### ISSUE TITLE
How to deploy distributed agents with TF Agent

### ISSUE THREAD

**Issue by Vetti420** [tensorflow/agents#956]
Hi, I am looking for something similar to what RLlib does in Tensorflow eco system. Since Rllib decided to ditch TF, and I am not sure about moving to PyTorch, i am looking for something that I can port here. I am using APPO feature of RLlib that deployes these agents across systems. And I have one trainer that collects the experiences from all these agents (like work while training mode, after it is trained offline for few days) and optimizes the policy. When restart, the model is loaded and these "working" agents are spun in multiple machines for inference only.

How do we do that with TF Agents. I tried looking, but did not find any example.

Thanks in Advance.
